src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeRequestHelper.java

```diff
@@ -20,7 +20,7 @@
 import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.HomeOfficeResponseException;
 
 @Service
-public class HomeOfficeApiUtil {
+public class HomeOfficeRequestHelper {
```

Comment:
This is still a bad name for a class.  We are trying to follow good OO practices so each class should have a single purpose.  http://ralin.io/blog/oop-anti-patterns-utility-or-helper-classes.html

Reply:
I created this class so I do not have to repeat the code for both api. It has a single purpose, which is creating Api request structure (like MessageHeader), generate correlation ID

Reply:
Renamed to HomeOfficeMessageHeaderHelper which helps create this json request
"messageHeader": {
        "consumer": {
            "code": "HMCTS",
            "description": "HM Courts and Tribunal Service"
        },
        "correlationId": "ABC2344BCED2234EA",
        "eventDateTime": "2020-04-26T17:32:28Z"
    }

Reply:
Also, the methods to extract response will get removed in the PR #77 
src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeRequestHelper.java